sources/newRif/rif_gemara_matcher_masoret.py

```diff
@@ -0,0 +1,69 @@
+import re
+import os
+import csv
+import copy
+import django
+django.setup()
+from functools import partial
+from sefaria.model import *
+from rif_utils import path, remove_metadata, tags_map, get_hebrew_masechet, hebrewplus, netlen2, unite_ref
+from research.mesorat_hashas_sefaria.mesorat_hashas import ParallelMatcher
+from sefaria.system.exceptions import InputError
+
+def base_tokenizer(string, masechet):
+    return remove_metadata(string, masechet).split()
+
+for masechet in list(tags_map):
+    heb_masechet = get_hebrew_masechet(masechet)
+    data = []
+    report = []
+    current_ref = 'דף ב.'
+    splitted_data = {current_ref: []}
+    matcher = ParallelMatcher(partial(base_tokenizer, masechet=masechet), verbose=False)
+
+    for root, dirs, files in os.walk(os.getcwd()+'/rif_csv'):
+        for file in files:
+            if masechet in file:
+                with open(path+'/rif_csv/'+file, encoding='utf-8', newline='') as file:
+                    data = list(csv.DictReader(file))
+
+    ref_tag = tags_map[masechet]['gemara_refs']
+    for row in data:
+        if row['content'].replace('\ufeff', '') == '': continue

```

Comment:
calling `str.replace` doesn't actually change the value stored at `row['content']`. I usually remove this character as well, but I would recommend either capturing the change or skipping it.